### PR TITLE
add draft of how to read

### PR DESCRIPTION
**Pull request by alunduil**

### CODE REVIEW

**Comment by jobsolete** [comment]
> I couple short reading bursts of an individual book with practised recall before continuing a book I read in a previous session

This sentence wouldn't strike me as terribly clear without my background understanding of your methodologies. I think it will stand up fine so long as it is combined with an example later in the article that is referenced here. Perhaps something like:

> I couple short reading bursts of an individual book with practised recall before continuing a book I read in a previous session (see example in [section]).

Followed later by an example in the previously referred to section outlining what this looks like.

**Comment by alunduil** [comment]
@OBrienCommaJosh I think I've added an example explaining what you're asking for but let me know if it's too muddled.

File: posts/2018-01-07-how-to-read.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: How to Read
```
Comment (perniferuse) [inline posts/2018-01-07-how-to-read.md]: Change to 'How I Read' or 'Read Like Me' or 'A Different Method for Reading' etc

File: posts/2018-01-07-how-to-read.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+---
+
+I've been told the rate and retention I have while reading is abnormal to say
+the least.  Others have been interested in trying my reading methodology and
+seeing if it would help them read better.  As a result, I've been making an
```
Comment (perniferuse) [inline posts/2018-01-07-how-to-read.md]: Remove 'read better' and replace with 'comprehend and retain more of what is read.'

File: posts/2018-01-07-how-to-read.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+attempt to codify how I read so that others can try it for themselves.
+
+The first idiosyncracy to my methodology is the concurrency I utilize while
```
Comment (perniferuse) [inline posts/2018-01-07-how-to-read.md]: I would delete the first line. It is redundant.

File: posts/2018-01-07-how-to-read.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+reading books.  Since college I've always read around five to ten books
+concurrently.  Generally, this is received with awe and intrigue about how I'm
+able to keep them separate or retain any number of them let alone one.
+
+What I've found over the years is that by reducing the amount of information
```
Comment (perniferuse) [inline posts/2018-01-07-how-to-read.md]: Change this sentence to: Over the years I've found that reducing the amount of information consumed on a topic in a given sitting (subsequently?)  increases the ability to let that knowledge settle into long-term memory (references).

File: posts/2018-01-07-how-to-read.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+consumed on a topic in a given sitting, it increases the ability to let that
+knowledge settle into long-term memory (references).  Thus, one of the ways to
+increase concurrency is to read a section or chapter of one book and then read a
+section or chapter of the next and repeat until done reading for the session.
+
+I couple short reading bursts of an individual book with practised recall before
```
Comment (perniferuse) [inline posts/2018-01-07-how-to-read.md]: Start paragraph with 'Simply put I couple short reading bursts...etc'

File: posts/2018-01-07-how-to-read.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+Elements", and "Abstract Algebra".  In my next reading session, I would take a
+moment before starting to recall the section or chapter I previously read of
+"Type-Driven Development with Idris".  Then I would read the next section or
+chapter before doing the same recall and reading of "The Thirteen Books of the
+Elements".
+
+By re-establishing reading context without re-reading the previous section
+before continuing a book, I ensure that I can keep the contents of a book
+relevant to some mnemonic.  Usually the author, title, or cover if not all
+three.
```
Comment (perniferuse) [inline posts/2018-01-07-how-to-read.md]: The last sentence is a sentence fragment. Remove the period after mnemonic and lowercase usually.

File: posts/2018-01-07-how-to-read.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+The second idiosyncracy to my methodoloy is the way I sort my books and order my
+reading.  I sort books I am currently reading by a quality I've been calling
+readability.  Loosely readability is a multiple of the approximate word density,
```
Comment (perniferuse) [inline posts/2018-01-07-how-to-read.md]: Add comma after Loosely.

File: posts/2018-01-07-how-to-read.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+age (time since added to the reading list), and completion.  This sorts my
```
Comment (perniferuse) [inline posts/2018-01-07-how-to-read.md]: After reading this paragraph, I have no idea that you measure the font to determine word density but that may be a failing of mine, not necessarily others. I also question what completion is. Is that how many words are left? Pages?

File: posts/2018-01-07-how-to-read.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+reading stack by how quickly I will finish books.  I add age to ensure that
+books aren't stuck at the bottom of the stack forever.
+
+Formally, I currently define readability as:
+
+```
+(Area of Page * Pages Remaining) / (Font Size * Completion Percentage * Age)
+```
```
Comment (perniferuse) [inline posts/2018-01-07-how-to-read.md]: Please add a conclusion to this post. A call to action is usually preferred such as "I'm interested to see if this method works for you." or something similar.

**Review by perniferuse — CHANGES_REQUESTED** [review]

File: posts/2018-01-07-how-to-read.md (hunk 8 above)
Comment (alunduil) [reply posts/2018-01-07-how-to-read.md]: Perhaps a quicker reference to the formal definition?  This is meant to be the rough idea whereas the equation below is supposed to be the details.

**Review by perniferuse — APPROVED** [review]

**Pull request merged**